bigquery-storage/api/BigQueryStorage.Samples/AppendRowsPending.cs

```diff
@@ -0,0 +1,154 @@
+            while (await appendRowResults.MoveNextAsync())
+            {
+                AppendRowsResponse responseItem = appendRowResults.Current;
+                // Do something with responses
+                Console.WriteLine($"Response Append Result is {responseItem.AppendResult}");
+            }
+            // The response stream has completed
+        });
+
+        // List of records to be appended in the table
+        // In Production scenario, these records would be pas
+        List<CustomerRecord> records = new List<CustomerRecord>
+        {
+            new CustomerRecord { CustomerNumber = 1, CustomerName = "Alice"},
+            new CustomerRecord { CustomerNumber = 2, CustomerName = "Bob"}
+        };
+
+        // Create a batch of row data by appending proto2 serialized bytes to the
+        // SerializedRows repeated field.
+        ProtoData protoData = new ProtoData
+        {
+            // This is a TEMPORARY WORKAROUND via extension method using reflection till the feature request
+            // and PR for "ToProto()" in Protobuf library gets released.
+            WriterSchema = new ProtoSchema { ProtoDescriptor = CustomerRecord.Descriptor.ToProto() },
+            Rows = new ProtoRows { SerializedRows = { records.Select(r => r.ToByteString()) } }
+        };
+
+        // Initialize the append row request
+        // Set an offset to allow resuming this stream if the connection breaks.
+        // In Production code, you may want to keep track of which requests the server has acknowledged
+        // and resume the stream at the first non-acknowledged message. If the server has already
+        // processed a message with that offset, it will return an ALREADY_EXISTS
+        // error, which can be safely ignored.
+        // The first request must always have an offset of 0.
+        AppendRowsRequest appendRowRequest = new AppendRowsRequest
+        {
+            WriteStreamAsWriteStreamName = stream.WriteStreamName,
+            Offset = 0L,
+            ProtoRows = protoData
+        };
+
+        // Stream a request to the server
+        await rowAppender.WriteAsync(appendRowRequest);
+
+        protoData = new ProtoData
+        {
+            Rows = new ProtoRows { SerializedRows = { new CustomerRecord { CustomerNumber = 3, CustomerName = "Charles" }.ToByteString() } }
+        };
+
+        // Since this is the second request, you only need to include the row data.
+        // The name of the stream and protocol buffers DESCRIPTOR is only needed in
+        // the first request.
+        appendRowRequest = new AppendRowsRequest
+        {
+            // Offset must equal the number of rows that were previously sent.
+            // If Offset is not present, the write is performed at the current end of stream. 
+            Offset = 2L,
+            ProtoRows = protoData
+        };
+
+        await rowAppender.WriteAsync(appendRowRequest);
+
+        // Complete writing requests to the stream
```

Comment:
Nit: I'd add a period here and to the second sentence in the statement below this one. Sentence fragments are okay in some cases, but when you're already writing a full sentence, I think it makes sense to end it with a period - particularly when there's a full sentence before it.

Reply:
Done.